pkg/sources/k8sevents/k8s_events.go

```diff
@@ -76,6 +76,10 @@ func (t *K8SEventsEventSource) StartFeed(trigger sources.EventTrigger, route str
 
 	// Just generate a random UUID as a subscriptionName
 	uuid, err := uuid.NewRandom()
+	if err != nil {
+		glog.Infof("Failed to create new random uuid: %v", err)
+		return nil, err
+	}
```

Comment:
Not part of this PR, but FYI we should use generateName for this. That requires a signature change to `createWatcher` to return the object or name generated by the apiserver.